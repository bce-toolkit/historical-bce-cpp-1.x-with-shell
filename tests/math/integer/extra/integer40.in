35797020546576589777834660877992775527405799568262325486268322402802051621838484774619874499940085932867124269996384536354996292395373174327360965626116677298782208754
55846621129872402604974226169099722530122547860841072514716
