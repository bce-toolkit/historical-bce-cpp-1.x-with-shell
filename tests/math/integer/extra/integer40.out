35797020546576589777834660877992775527405799568262325486268322402802051621838484774619874499940085932867124325843005666227398897369599343427083495748664538139854723470
35797020546576589777834660877992775527405799568262325486268322402802051621838484774619874499940085932867124214149763406482593687421147005227638435503568816457709694038
1999142644042920721278735902457359135093690539835819927134169795585872879349032565276773449487868790847448077216416028739188677994966804107804578849879819547198756109875057981350200412990970909278229357130188988841127649023864
640988117496489589179013014901911210499905493545853841539409025103036681124904287676091570231395010551415734
49903367766015213976426228068248511999916872852089433267210
0
1
0
0
1
1
